libarchive/ffi.py

```diff
@@ -125,7 +125,7 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_int], c_int)
+ffi('entry_set_size', [c_archive_entry_p], c_longlong)
```

Comment:
The way we test things in this library is mostly by creating archives and then reading them back. Comparing the two data sets allows us to detect bugs. For example, if you write an entry size of 42 and you read back a size of 0 or 10000, then you know that something's wrong.